test/test_jit_cuda_fuser.py

```diff
@@ -259,8 +266,8 @@ def t(x, y, z, q):
         self.assertEqual(o, jit_o)
         self.assertGraphContains(t_jit.graph_for(x, y, z, q), FUSION_GUARD)
 
-    @unittest.skipIf(is_pre_volta(), "reduction not supported in pre volta device")
```

Comment:
Note that I swapped these skipIf statements arround, because if RUN_CUDA is false, we would like to report 'requires CUDA' as a reason for skipping, not 'reduction not supported in pre volta device'.